sql/chars.sql

```diff
@@ -6,6 +6,7 @@ DROP TABLE IF EXISTS `chars`;
 CREATE TABLE `chars` (
   `charid` int(10) unsigned NOT NULL,
   `accid` int(10) unsigned NOT NULL,
+  `original_accid` int(10) unsigned NOT NULL DEFAULT '0',
```

Comment:
Since its a char table, we'll want a migration in dbtool to add the new column

Reply:
Added the migration script